src/applications/disability-benefits/all-claims/tests/data/maximal-test.json

```diff
@@ -6,9 +6,6 @@
     },
     "standardClaim": false,
     "view:fdcWarning": {},
-    "waiveTrainingPay": true,
```

Comment:
:thinking: Should all this stuff in here be removed? Is this all because the pages are skipped since the user has rated disabilities and have thus already answered these questions?

Reply:
Yeah, that's exactly right. They'll never see these pages anymore when they have any rated disabilities coming through in prefill.